_topic_map.yml

```diff
@@ -1207,6 +1207,13 @@ Topics:
     File: odo-architecture
   - Name: Installing odo
     File: installing-odo
+  - Name: Using odo in a restricted environment  
+    Dir: using_odo_in_a_restricted_environment
+    Topics: 
+    - Name: Pushing the `odo init` image to the restricted cluster registry
```

Comment:
The backticks are rendering in the TOC. You should remove them because I don't think they're accepted in the TOC.